src/test/java/com/nonononoki/alovoa/html/UserProfileResourceTest.java

```diff
@@ -97,4 +108,17 @@ void test() throws Exception {
 		String encodedId = UserDto.encodeId(user.getId(), textEncryptor);
 		userProfileResource.profileView(encodedId);
 	}
+
+	@Test
+	void profileView__validUser__profileDisplayed() throws Exception {
+		Mockito.when(authService.getCurrentUser()).thenReturn(testUsers.get(0));
```

Comment:
This is a bit silly as we really don't need to mock the `AuthService` once we actually have working auth in tests.
If we don't mock the `AuthService` then the ApplicationContext can't be reused though.
But I think the `@AutoConfigureMockMvc` will change the ApplicationContext hash anyways, since it adds the `MockMvc` to the context.
My suggestion would be to move all the `@SpringBootTest` classes to this pattern.